### PR TITLE
chore(blog/docs): use `postcss-import` to import css variables styles

### PR DESCRIPTION
**Pull request by B2o5T**
also, we can share `code-blocks` styles to avoid code duplications but in another PR

### CODE REVIEW

File: packages/nextra-theme-docs/src/styles.css [hunk 1]
```diff
@@ -1,4 +1,15 @@
...
+@import 'tailwindcss/utilities';
+@import './variables.css';
+
+:root {
+  --nextra-primary-hue: 212deg;
+}
+
+.dark {
+  --nextra-primary-hue: 204deg;
+}
```
Comment (B2o5T) [inline packages/nextra-theme-docs/src/styles.css:12]: only these variables are included in next-theme-docs

File: packages/nextra-theme-docs/package.json [hunk 2]
```diff
@@ -61,7 +61,6 @@
   "devDependencies": {
     "@types/flexsearch": "^0.7.2",
     "@types/react": "^17.0.39",
-    "autoprefixer": "^10.2.6",
```
Comment (B2o5T) [inline packages/nextra-theme-docs/package.json:64]: don't need in docs since `autoprefixer` is already installed in the root `package.json`

**Comment by B2o5T** [comment]
@shuding can I merge this one? and I'll extract code-styles in follow PR

**Comment by shuding** [comment]
Yeah sounds good to ship it 👍❤️

**Pull request merged**